### ISSUE TITLE
Multiple exchanges arbitrage seems impossible?

### ISSUE THREAD

**Issue by zgayjjf** [wardbradt/peregrine#65]
How can a trader make a profit across multiple exchanges? 
He should withdraw a coin from an exchange and deposit in another exchange. If there are more than 2 exchanges, this would drastically increase the whole time.

Even if we have assets in all the exchanges already, we would notice that when we sold out all the base coin in one exchange, we will never have the chance to buy them back.

So, it is possible to make a profit across multiple exchanges?

**Comment by noahdahlman** [comment]
Instead of thinking of your exchange accounts as separate, it may be more intuitive to think of them as being the same portfolio. 
![arbitrage](https://user-images.githubusercontent.com/60643920/89745166-e5d71000-da7f-11ea-9829-3db1b4e55132.png)
As long as the total value of your portfolio increases, you are making a profit, even if your are making transactions on separate accounts.

**Comment by robertsmaoui** [comment]
noahdahlman  not True !! because the crypto bought in first exchange cannot be sold in the second exchange, the crypto is not available (balance 0) !!

**Comment by noahdahlman** [comment]
Yes, in this case, you would need to be holding balances on both exchanges, and in some cases, you can cycle funds by buying and transferring them before selling. I've seen some arbitrages last for hours, especially on smaller exchanges with low liquidity.
